components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt

```diff
@@ -184,8 +184,10 @@ class FirefoxAccount internal constructor(
         }
     }
 
-    override fun getTokenServerEndpointURL(): String {
-        return inner.getTokenServerEndpointURL()
+    override suspend fun getTokenServerEndpointURL() = withContext(scope.coroutineContext) {
```

Comment:
Curious how this didn't fail before? Was it wrapped in an `handleFxaExceptions` and we lost that? Or are we calling it more often now?

Reply:
Still investigating this. It's neither of those, I'll let you know once I sort this out. Maybe it _was_ failing and we didn't notice, somehow..?